tailcfg/tailcfg.go

```diff
@@ -142,6 +142,7 @@ type CapabilityVersion int
 //   - 99: 2024-06-14: Client understands NodeAttrDisableLocalDNSOverrideViaNRPT
 //   - 100: 2024-06-18: Client supports filtertype.Match.SrcCaps (issue #12542)
 //   - 101: 2024-07-01: Client supports SSH agent forwarding when handling connections with /bin/su
+//   - 102: 2024-07-04: Client supports NodeAttrCaptivePortalDetection
 const CurrentCapabilityVersion CapabilityVersion = 101
```

Comment:
forgot to set 102 here so I assumed a test failed to tell you the comment doesn't match this line? (we did this so many times we added a test for it)